assets/js/hooks/useMonitorInternetConnection.test.js

```diff
@@ -127,43 +139,4 @@ describe( 'useMonitorInternetConnection', () => {
 		expect( store.getState().isOnline ).toBe( true );
 		expect( fetchMock ).toHaveFetched( healthCheckEndpoint );
 	} );
-
-	it( 'should check online status in correct intervals', async () => {
```

Comment:
Hmm, I would have thought you could keep this test and rewrite it to use Jest [timer mocks](https://jestjs.io/docs/timer-mocks)...

Reply:
Since I removed the optional parameter for `intervalTime`, default timeout values are too hight to be able to test them, as we have 5000ms timeout limit, so `advanceTimersByTime` would timeout due to this.

Reply:
Using the timer mocks decouples the tests from the real system time, allowing us to test arbitrarily high delays - you should be OK here. For example:

https://github.com/google/site-kit-wp/blob/274839bd27eebaad2d9c4a689beb4043449e80b1/assets/js/googlesitekit/datastore/user/surveys.test.js#L238-L244